Exercises/2.h

```diff
@@ -16,18 +16,22 @@ class PathDecomposition {
 	Thrown when path-width decomposition is incorrect
 	*/
 	private:
-		std::string _msg = "";
 	public:
+		int type = 0; // 0 or 1
```

Comment:
please give these fields meaningful names and make it possible to set them in the constructor. Also, they should start with subscript _. Make an enum for types so the values also have meaningful names.